agent/internal/container/container.go

```diff
@@ -393,11 +393,15 @@ func (c *Container) shimDockerEvents() events.Publisher[docker.Event] {
 	return events.FuncPublisher[docker.Event](func(ctx context.Context, e docker.Event) error {
 		switch {
 		case e.Log != nil:
+			source := "agent"
+			agentID := "agent-id" // Place holder -- what should we put here?
```

Comment:
probably should be agent id. i think you could enrich in `agent.go` instead and it is available there.